src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+html {
+  box-sizing: border-box;
+}
+
+:root {
+  --blue-color: #00acdc;
+  --black-color: #000;
```

Comment:
Try not to use color name in variables names
Example of better name: "--main-color", "--hover-color"...